src/pages/index.js

```diff
@@ -109,21 +181,97 @@ function closeModal(modal) {
   modal.removeEventListener("mousedown", handleOverLayClose);
 }
 
+function handleLike(evt, id) {
+  const isLiked = evt.target.classList.contains("card__like-btn_liked");
+
+  api
+    .handleLikeStatus(id, isLiked)
+    .then((card) => {
+      if (card) {
+        evt.target.classList.toggle("card__like-btn_liked");
+        localStorage.setItem(`card-${id}-isLiked`, !isLiked);
```

Comment:
Please, don't use `localStorage` for liking because everything is inside `data.isLiked`